checks/core/cloudfront_age.py

```diff
@@ -8,6 +8,7 @@
 
 
 EXPOSED_PARAMETERS = ["url", "max_age"]
+DEFAULT_PLOT = "."
```

Comment:
You mean like ability to define a function per check module?

Reply:
Yes, each module could offere a function to extract the data instead of the `DEFAULT_PLOT` string. But we are porbably not going to need it, and if we do, we can still change it. I just mentioned an idea I had when I read the code, no need to change anything.

Reply:
Let's keep it this way, it serves as a kind of «interface» between the check and the app :)